addons/account/static/src/js/vat_activity.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+const { Component } = owl;
+
+export class JournalDashboardActivity extends Component {
```

Comment:
File is named `vat_activity` but the class name is more generic (I think it should be more generic, AFAIK it is also used for other things i.e. for fiduciaries)

Reply:
rename to JournalDashBoardActivity